dashboard/src/pages/packages/[id]/index.vue

```diff
@@ -68,7 +70,39 @@ const createAipWorkflow = computed(
 
     <div v-if="authStore.checkAttributes(['package:listActions'])">
       <div class="d-flex">
-        <h2 class="mb-0">Preservation actions</h2>
+        <h2 class="mb-0">
+          Preservation actions
+          <a
+            data-bs-toggle="collapse"
+            href="#presActionInfo"
+            role="button"
+            aria-expanded="false"
+            aria-controls="presActionInfo"
+            ><IconInfoStandardSolid style="font-size: 0.6em" aria-hidden="true"
```

Comment:
You should be able to set the size directly in the import, like we do in the sidebar component:

https://github.com/artefactual-sdps/enduro/blob/main/dashboard/src/components/Sidebar.vue#L8